axelor-project/src/main/java/com/axelor/apps/project/web/ProjectTaskController.java

```diff
@@ -338,4 +338,22 @@ public void updateBudgetedTime(ActionRequest request, ActionResponse response)
     response.setValues(
         Beans.get(ProjectTaskGroupService.class).updateBudgetedTime(projectTask, oldTimeUnit));
   }
+
+  @ErrorException
+  public void statusOnSelect(ActionRequest request, ActionResponse response) {
+    ProjectTask projectTask = request.getContext().asType(ProjectTask.class);
+    Project project = projectTask.getProject();
+
+    Set<TaskStatus> taskStatusSet =
+        Beans.get(TaskStatusToolService.class).getTaskStatusSet(project, projectTask);
+    String filter = "self.id IN (0)";
```

Comment:
You can use the IdListString here also no ? If the list is empty, it send only a 0